scrapers/WAPdB.py

```diff
@@ -0,0 +1,222 @@
+            height = height_node.text.split(height_prefix)[1].split()[0]
+            if height != 'unknown':
+                performer['height'] = height
+    if image_node := soup.find('div', {'id': 'casting-profil-preview'}):
+        image_url = image_node.find('img', {'itemprop': 'image'}).attrs['src']
+        if '/WAPdB-img/par-defaut/' not in image_url:
+            performer['image'] = f'http://warashi-asian-pornstars.fr{image_url}'
+    return performer
+
+
+def scrape_full_profile(soup, url):
+    performer = {}
+    measurements_prefix = 'measurements: '
+    activity_prefix = 'porn/AV activity: '
+
+    if alphabet_name := soup.find('span', {'itemprop': 'name'}):
+        alphabet_name = alphabet_name.text
+
+    japanese_name = None
+    if additional_name := soup.find('span', {'itemprop': 'additionalName'}):
+        japanese_name = additional_name.text
+    performer['name'] = process_name(alphabet_name)
+    performer['url'] = url
+    if gender := get_gender(url):
+        performer['gender'] = gender
+    if gender_node := soup.find('meta', {'property': 'og:gender'}):
+        performer['gender'] = gender_node.attrs['content']
+    if twitter_node := soup.find(text='official Twitter'):
+        performer['twitter'] = twitter_node.parent.attrs['href']
+    if birthday_node := soup.find('time', {'itemprop': 'birthDate'}):
+        performer['birthdate'] = birthday_node.attrs['content']
+    if height_node := soup.find('p', {'itemprop': 'height'}):
+        if height_value_node := height_node.find('span', {'itemprop': 'value'}):
+            performer['height'] = height_value_node.text
+    if weight_node := soup.find('p', {'itemprop': 'weight'}):
+        if weight_value_node := weight_node.find('span', {'itemprop': 'value'}):
+            performer['Weight'] = weight_value_node.text
+    if measurements_node := soup.find(text=lambda t: measurements_prefix in str(t)):
+        measurements = measurements_node.text.split(measurements_prefix)[1]
+        if measurements != 'unknown':
+            performer['measurements'] = measurements_node.text.split(measurements_prefix)[1]
+    if activity_node := soup.find(text=lambda t: activity_prefix in str(t)):
+        performer['career_length'] = activity_node.text.split(activity_prefix)[1].strip()
+
+    if image_container_node := soup.find('div', {'id': 'pornostar-profil-photos-0'}):
+        if image_node := image_container_node.find('img', {'itemprop': 'image'}):
+            image_url = image_node.attrs['src']
+            if '/WAPdB-img/par-defaut/' not in image_url:
+                performer['image'] = f'http://warashi-asian-pornstars.fr{image_url}'
+
+    if len(country_nodes := soup.find_all('span', {'itemprop': 'addressCountry'})) > 1:
+        country = country_nodes[1].text
+        performer['country'] = country
+        if country == 'Japan':
+            performer['ethnicity'] = 'Japanese'
```

Comment:
for ethnicity we use `asian` 
i think its safe to use it as a fixed value insteas as all the performers in the site are asians ?

Reply:
Wasn't sure what the convention was as "asian" where I'm from means people from the Indian subcontinent. There are some American and French performers on the site so it's not safe to make it a static value.

Reply:
Yeah the `asian performers database` in the title made me think that...